src/utils.ts

```diff
@@ -222,3 +222,16 @@ export function getGlobalCSSVariable(name: string) {
     `--${name}`,
   );
 }
+
+export const isServer = () => typeof window === "undefined";
+
+export const getPixelRatio = (): number =>
+  isServer() ? 1 : window.devicePixelRatio;
```

Comment:
Same here, can you check if devicePixelRatio exists in the window object (and the window object exists), if not, return 1